modules/common/react/lib/theming/createCanvasTheme.ts

```diff
@@ -19,8 +20,7 @@ enum ColorDirection {
 }
 
 function shiftColor(hexColor: string, direction: ColorDirection) {
-  const canvasColor =
-    colors[Object.keys(colors).filter(color => colors[color] === hexColor) as CanvasColor];
+  const canvasColor = findKey(colors, hexColor);
```

Comment:
I don't think the second argument is right here, it is either a function or an iteratee shorthand. In this case you're asking to find the key in which a `hexColor` is `true`.

Can you add a quick test for this? This code is uncovered so that's why this isn't failing on this change. I think testing to see what something like `colors.soap300` would output would be good enough for now.

Ideally I would like to cover the three functions as individual units in this file, but I can open up an issue to take care of that for later.

<img width="289" alt="Screen Shot 2020-06-15 at 9 48 10 AM" src="https://user-images.githubusercontent.com/14299381/84684360-6993d680-aeed-11ea-87c2-66584ca08c21.png">


```suggestion
  const canvasColor = findKey(colors, color => color === hexColor);
```

Reply:
Hah - you are correct. I thought the tests would have caught it so I assumed this was right. Good call 👍  Will update